parsec/core/fs/workspacefs/workspacefile.py

```diff
@@ -33,13 +33,13 @@ class WorkspaceFile:
     Keyword arguments:
     transactions -- object to use for file transactions.
     path -- relative path of the file.
-    mode -- the mode where the file have been opened. It needs to have exactly one of "awrx".
+    mode -- the mode where the file have been opened. It needs to have exactly one of "arwx".
     mode("a") -> append file.
-    mode("b") -> have to be used with one of "awrx" mode, file will be write/read as bytes instead of string.
+    mode("b") -> have to be used with one of "arwx" mode, file will be write/read as bytes instead of string.
     mode("w") -> write file. If the file doesn't exist, create one.
     mode("r") -> read file.
     mode("x") -> If the file doesn't exist, create one, else, raise an Error. File will be opened in write mode.
-    mode("+") -> have to be used with one of "awrx" mode, file will be in read/write mode.
+    mode("+") -> have to be used with one of "arwx" mode, file will be in read/write mode.
```

Comment:
and here too.

Reply:
https://github.com/Scille/parsec-cloud/pull/3772#discussion_r1049637877